lib/settings/transformers/v1Settings.js

```diff
@@ -0,0 +1,27 @@
+const _ = require('lodash')
+const consts = require('../lib/consts')
+
+class V1Settings {
+  static transform (Settings) {
+    let transformedSettings = _.cloneDeep(Settings)
+    setSettingsDefault(transformedSettings)
+    return transformedSettings
+  }
+}
+
+const checkAndSetDefault = (ruleSet, defaultValue) => {
+  if (ruleSet === undefined) {
+    return defaultValue
+  }
+  return ruleSet
+}
+
+const setSettingsDefault = (Settings) => {
+  let mergeableSettings = Settings.mergeable
+
+  mergeableSettings.use_config_from_pull_request = checkAndSetDefault(mergeableSettings.use_config_from_pull_request, consts.DEFAULT_USE_CONFIG_FROM_PULL_REQUEST)
+  mergeableSettings.use_config_cache = checkAndSetDefault(mergeableSettings.use_config_cache, consts.DEFAULT_USE_CONFIG_CACHE)
+  mergeableSettings.use_org_as_default_config = checkAndSetDefault(mergeableSettings.use_org_as_default_config, consts.DEFAULT_USE_ORG_AS_DEFAULT_CONFIG)
```

Comment:
```suggestion
  mergeableSettings.useConfigFromPullRequest = checkAndSetDefault(mergeableSettings.useConfigFromPullRequest, consts.DEFAULT_USE_CONFIG_FROM_PULL_REQUEST)
  mergeableSettings.useConfigCache = checkAndSetDefault(mergeableSettings.useConfigCache, consts.DEFAULT_USE_CONFIG_CACHE)
  mergeableSettings.useOrgAsDefaultConfig = checkAndSetDefault(mergeableSettings.useOrgAsDefaultConfig, consts.DEFAULT_USE_ORG_AS_DEFAULT_CONFIG)
```

use camel case instead?

Reply:
They are coming from `mergeable.settings.yml` and it's not good to use camel variables in yaml (because we never used)

like:

```
- do: description
        must_exclude (HERE):
```